### ISSUE TITLE
Drop column when after loading CSV

### ISSUE THREAD

**Issue by koslambrou** [acowley/Frames#137]
Is it currently possible to drop specific columns after loading a CSV file in Frames?

**Comment by acowley** [comment]
Yes, see this [tutorial example](https://github.com/acowley/Frames/blob/c025810efab7f32fe649f59e7573eb4a71b8e78f/demo/Tutorial.hs#L318). I find it a bit awkward to work with if you end up doing a lot of insertions and deletions; at the very least you will want to define a `type` synonym for the row with the deleted columns.
